### PR TITLE
Proposed fix for "In a DBGrid when you move a column at the right end, the grid is repositioned to the start left"

### PR DESCRIPTION
**Pull request by MHumm**
Fix for Mantis issue 6540:
[http://issuetracker.delphi-jedi.org/view.php?id=6540](url)

### CODE REVIEW

File: jvcl/run/JvDBGrid.pas [hunk 1]
```diff
@@ -2819,6 +2819,7 @@ procedure TJvDBGrid.MouseUp(Button: TMouseButton; Shift: TShiftState;
   ACol: Longint;
   DoClick: Boolean;
   ALeftCol: Integer;
+  SI: TScrollInfo;
```
Comment (obones) [inline jvcl/run/JvDBGrid.pas]: This variable should really be named `OriginalScrollInfo`

File: jvcl/run/JvDBGrid.pas [hunk 2]
```diff
@@ -2872,7 +2873,18 @@ procedure TJvDBGrid.MouseUp(Button: TMouseButton; Shift: TShiftState;
         ColWidths[FSizingIndex]);
   end
   else
-    inherited MouseUp(Button, Shift, X, Y);
+  begin
+    SI.cbSize := SizeOf(SI); //
```
Comment (obones) [inline jvcl/run/JvDBGrid.pas]: What's the point of the empty comment?

File: jvcl/run/JvDBGrid.pas [hunk 3]
```diff
@@ -2872,7 +2873,18 @@ procedure TJvDBGrid.MouseUp(Button: TMouseButton; Shift: TShiftState;
...
+    SI.fMask := SIF_POS; // Store scrollbar position
```
Comment (obones) [inline jvcl/run/JvDBGrid.pas]: This comment should really be on the next line

**Comment by MHumm** [comment]
Superseded by #116

**Pull request closed without merging**